e2e/cypress/support/api_commands.js

```diff
@@ -723,4 +723,15 @@ Cypress.Commands.add('demoteUser', (userId) => {
     //Demote Regular Member to Guest User
     const baseUrl = Cypress.config('baseUrl');
     cy.externalRequest({user: users.sysadmin, method: 'post', baseUrl, path: `users/${userId}/demote`});
-});
\ No newline at end of file
+});
+
+/**
+ * Promote a Guest to a Member directly via API
+ * @param {String} userId - The user ID
+ * All parameter required
+ */
+Cypress.Commands.add('promoteUser', (userId) => {
+    //Demote Regular Member to Guest User
```

Comment:
The comment is wrong

Reply:
Thanks @jespino. I have corrected the comment.